fun a(){
    print 1;
}

fun main(){
    y = (2 + 3;
    x = print 0; #the previous line is missing a paren
    #If you run this as `make errors.S` you'll see the error message
    #obviously you can't expect this test to "pass" so idk what to do about that
}